### PR TITLE
73 Login

### LINKED ISSUE #73: Login
## Description 
As a Po I want a user to be able to login into the Seaqull website.

 **Describe the solution you'd like**
Create a login page

 **Describe alternatives you've considered**
Does not apply

 **Additional context**

 "Image of the design will be added here when it's done"

 **D.O.D.**
Login Page should be according to design.

### PR DESCRIPTION
**Pull request by MiguelFieira**
closes #73

 **What is it supposed to do**

 **How can we test your branch**

 **What broweser have been checked**
- [x] Chrome
- [x] Firefox
- [x] Opera
- [ ] Internet Explorer
- [ ] Edge
- [ ] Safari

### CODE REVIEW

File: src/components/input/index.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    return (
+      <div className={styles.username}>
+        <label for={name} className={styles.label}>{title}</label><br />
+        <input type={type} id={name} name={name} className={styles.input} />
+      </div>
+    )
+  }
+}
+
+export default Input
```
Comment (MiguelFieira) [inline src/components/input/index.js]: endline

File: src/components/prompt/index.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+            </div>
+          </form>
+          <div className={styles.image} />
+        </div>
+      </div>
+    )
+  }
+}
+
+export default Prompt
```
Comment (MiguelFieira) [inline src/components/prompt/index.js]: endline

File: src/components/prompt/prompt.scss [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+		height: 100%;
+		max-width: 100%; 
+		padding: 0;
+	}
+}
+
+.text {
+	text-align: center;
+	font-weight: bold;
+	font-size: 2.3rem;
```
Comment (MiguelFieira) [inline src/components/prompt/prompt.scss]: 2rem looks more like the design

File: src/components/prompt/prompt.scss [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+	}
+}
+
+.image {
+	display: inline-block;
+	min-width: 110px;
+	margin-top: auto;
+	height: calc(110px * 1.65);
+	border-radius: 10px;
+	margin-left: 30px;
```
Comment (Samvdg) [inline src/components/prompt/prompt.scss]: please use margin shorthand

File: src/components/prompt/prompt.scss [hunk 5]
```diff
@@ -0,0 +1,149 @@
...
+	@include mobile-all-portrait {
+		word-break: break-all;
+		font-size: 2rem;
+		color: #fff;
+		font-weight: bold;
+	}
+}
+
+.input {
+	font-family: Poppins;
```
Comment (MiguelFieira) [inline src/components/prompt/prompt.scss]: Poppins should already be the default font so this line is useless

**Review by Samvdg — APPROVED** [review]

**Pull request closed without merging**